.github/workflows/all-checks.yaml

```diff
@@ -75,3 +75,46 @@ jobs:
         with:
           name: coverage-out
           path: ${{env.GOPATH}}/src/github.com/insolar/assured-ledger/ledger-core/v2/coverage.out
+  tests-on-windows:
+    name: windows-checks
+    runs-on: windows-latest
+    env: # we have to specify fake BUILD_DATE and BUILD_TIME here because git-date-time.sh doesn't work on Windows
+      GOPATH: D:/a/assured-ledger/assured-ledger/go
+      WORKDIR: D:/a/assured-ledger/assured-ledger/go/src/github.com/insolar/assured-ledger/ledger-core/v2
+      BUILD_DATE: 20200101
```

Comment:
"time /T" and "date /T" do not work as well?

Reply:
I don't know. As you might guess there are not many Windows users among us.